docs/hobbies.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
```

Comment:
First this is in the wrong folder. `/docs` is for documentation markdown files and images for those files. Also this pull request is only about registration form please remove this file from the pull request. You may submit it in another

Reply:
@stankasvecova please move it into html folder. 
And this part has nothing to do with registration, so please remove it from here and continue with it in different branch.

Reply:
Done